test/dev-app/models/test.lua

```diff
@@ -0,0 +1,19 @@
+local val = require "valua"
```

Comment:
Yes. This is the model file for ES documents. You have to create an instant of this model and then use it's methods for accessing ES. It has the "type" and "attributes" for the model. 

If you talking about the "valua" import, I will add support for checking attributes in the model similar to "model.lua".